docs/docs/keymaps/input-processors/behaviors.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Behaviors Input Processor
+sidebar_label: Behaviors
+---
+
+## Overview
+
+The behaviors input processor is used invoke standard behaviors when certain input events occur; most frequently this is used to trigger behaviors when certain mouse buttons are triggered by physical pointing devices.
+
+:::note
+
+The behaviors input processor is primarily intended for `INPUT_EV_KEY` type of events that have a binary on/off state, not vector types for relative or absolute movements.
```

Comment:
```suggestion
This input processor is primarily intended for `INPUT_EV_KEY` type of events that have a binary on/off state, not vector types for relative or absolute movements.
```
It was a bit repetitive since it mirrors the start of the previous paragraph.